java/spacewalk-java.changes

```diff
@@ -1,3 +1,4 @@
+- Add the possibility to filter notifications types through a configuration parameter: java.notifications_type_filter.
```

Comment:
Aren't you missing bsc#1111910 here?

If this was backported to any other versions, remember to add it if needed.

Reply:
Unless this was already ported to 3.2, this should not close SUSE/spacewalk#6114 as fixed.

Reply:
yes, it is missing :)  I was treating this as a feature and forgot it has a bug associated.